server/controllers/users.js

```diff
@@ -141,13 +145,34 @@ class Users {
     })
 
     .then((user) => {
-      if (user.length < 1)
+      if (user.length < 1) {
         return res.status(400).send({
           message: 'No users match the search criteria'
         });
-      return res.status(200).send(user);
+      }
+      return res.status(200).send({ message: 'Listing all the users that match the search criteria', user });
     })
     .catch(error => res.status(400).send(error));
   }
+
+  logout(req, res) {
```

Comment:
**Question** - This seems like a no-op to me. Any particular reason we have it here?